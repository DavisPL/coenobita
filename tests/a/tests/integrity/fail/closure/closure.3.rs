#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::tag({*}{*} fn({bin}{bin}, {bin}{bin}) -> {bin}{bin})]
    let f = if b::boolean() && c::boolean() {
        |x: i32, y: i32| x 
    } else {
        |x: i32, y: i32| y
    };

    #[cnbt::tag({bin}{bin})]
    let y = f(5, 6);
}