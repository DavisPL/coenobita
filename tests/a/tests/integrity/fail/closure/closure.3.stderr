error: expected {bin}{bin}, found {bin}{b,bin,c}
 --> tests/integrity/fail/closure/closure.3.rs:7:26
  |
7 |         |x: i32, y: i32| x
  |                          ^
